src/lib/firebase.js

```diff
@@ -5,13 +5,14 @@ import { getFirestore } from 'firebase/firestore';
 // Initialize Firebase.
 // These details will need to be replaced with the project specific env vars at the start of each new cohort.
 const firebaseConfig = {
-  apiKey: "AIzaSyApo-FwnsG_ISGLcsrFp9dwg4DjisoqS_s",
-  authDomain: "tcl-41-smart-shopping-list.firebaseapp.com",
-  projectId: "tcl-41-smart-shopping-list",
-  storageBucket: "tcl-41-smart-shopping-list.appspot.com",
-  messagingSenderId: "683972107350",
-  appId: "1:683972107350:web:80b848263dca925a386ff7"
+  apiKey: 'AIzaSyApo-FwnsG_ISGLcsrFp9dwg4DjisoqS_s',
+  authDomain: 'tcl-41-smart-shopping-list.firebaseapp.com',
+  projectId: 'tcl-41-smart-shopping-list',
+  storageBucket: 'tcl-41-smart-shopping-list.appspot.com',
+  messagingSenderId: '683972107350',
+  appId: '1:683972107350:web:80b848263dca925a386ff7',
 };
 
 export const app = initializeApp(firebaseConfig);
+//added 'app' inside getFirestore func
```

Comment:
I'm not sure if I understand this comment.

Reply:
when we were reading the documentation for Firebase we saw a lot of examples having
export const app = initializeApp(firebaseConfig);
export const db = getFirestore(app); << with app in there

so we commented that to remember we did or did not include it